homeassistant/components/fastdotcom/__init__.py

```diff
@@ -40,38 +35,59 @@
 )
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+async def async_setup_platform(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Fast.com component. (deprecated)."""
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": SOURCE_IMPORT},
+            data=config[DOMAIN],
+        )
+    )
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up the Fast.com component."""
-    conf = config[DOMAIN]
     data = hass.data[DOMAIN] = SpeedtestData(hass)
 
-    if not conf[CONF_MANUAL]:
-        async_track_time_interval(hass, data.update, conf[CONF_SCAN_INTERVAL])
+    async_track_time_interval(hass, data.update, timedelta(hours=DEFAULT_INTERVAL))
```

Comment:
We should stop this if the entry get's unloaded

Reply:
Fixed in the latest commit.